angular-workspace/projects/ni/nimble-angular/src/thirdparty/testing/matchers.ts

```diff
@@ -0,0 +1,237 @@
+/**
+ * [Nimble]
+ * Copied from https://github.com/angular/angular/blob/6070c9ddcff88d4ad4bcf73a2dd1874920661d93/packages/platform-browser/testing/src/matchers.ts
+ * with the following modifications:
+ * - Update imports
+ * - Comment out everything other than what is needed to use `toHaveText` matcher
```

Comment:
Can you mention why? Guessing to avoid polluting the jasmine scope with extra matchers only needed for thirdparty?

Reply:
I had added this file. More generally, I wanted to minimize impact to our codebase. For example, in the future, if we merge changes to the commented-out code, it will be immediately clear that it does not affect us.

Reply:
@m-akinc, you did add this file, but I believe I was actually the one that comment out most of the implementation 😄 

There were a lot of things in this file that were using private Angular APIs, and since we didn't need anything other than `toHaveText`, I didn't see the need to find alternatives to the private APIs. I also agree with the rationale that it's easy to tell that commented out code changing doesn't have an impact to our codebase.

Reply:
I've updated the comment.

Reply:
> @m-akinc, you did add this file, but I believe I was actually the one that comment out most of the implementation 😄

😳 Whoops! I assumed I had done it, and then gave the rationale I thought I used! 😂